osxarm64: 
```
CMake Error at /Users/runner/miniforge3/conda-bld/pivy_1657479386200/_build_env/share/cmake-3.23/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Python (missing: Interpreter) (found version "2.7.16")
```

@conda-forge/core any ideas?

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -8,7 +8,8 @@ fi
 
 PIVY_CPP_FLAGS='-std=c++1z '
 
-cmake -G "Ninja" \
+cmake ${CMAKE_ARGS} \
+    -G "Ninja" \
     -D CMAKE_BUILD_TYPE="Release" \
     -D CMAKE_INSTALL_PREFIX:FILEPATH=$PREFIX \
     -D Python_EXECUTABLE="$PYTHON" \

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -16,6 +16,8 @@ build:
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - {{ compiler("cxx") }}
     - {{ cdt('mesa-libgl-devel') }}      # [linux]
     - msinttypes      # [win]

